src/screens/BookmarkCategoryScreen.js

```diff
@@ -58,17 +59,21 @@ export const BookmarkCategoryScreen = ({ navigation }) => {
   // and toggling the bookmark status through the header
   if (bookmarks.length === 0) {
     return (
-      <Wrapper>
-        <RegularText>{texts.bookmarks.noBookmarksinCategory}</RegularText>
-      </Wrapper>
+      <WrapperWithOrientation style={styles.warningText}>
+        <Wrapper>
+          <RegularText>{texts.bookmarks.noBookmarksinCategory}</RegularText>
+        </Wrapper>
+      </WrapperWithOrientation>
     );
   }
 
   if (!data) {
     return (
-      <Wrapper>
-        <RegularText>{texts.errors.noData}</RegularText>
-      </Wrapper>
+      <WrapperWithOrientation style={styles.warningText}>
+        <Wrapper>
```

Comment:
this is adding an extra verticalPadding when landscape or big screen . To solve this we could have a condition using OrientationContext like you did in CardList : 
<img width="842" alt="Screenshot 2021-01-13 at 13 58 57" src="https://user-images.githubusercontent.com/43641321/104455478-9dc89600-55a7-11eb-8616-f96739bb9e4d.png">